charts/victoria-metrics-cluster/values.yaml

```diff
@@ -7,6 +7,17 @@
 ##
 clusterDomainSuffix: cluster.local
 
+rbac:
+  create: true
+  pspEnabled: true
+  namespaced: false
```

Comment:
is it better to remove this param and always create ClusterRoleBinding?

Reply:
I don't think so. For us for example we would like to have it contained to a namespace. It should also make it possible for "regular" users to install the chart. While creating ClusterRole(Binding)s require ClusterAdmin privileges I think.